cli/command/image/tree_test.go

```diff
@@ -126,6 +150,7 @@ func TestPrintImageTreeAnsiTty(t *testing.T) {
 				assert.Check(t, hasAnsi, "Output should contain ANSI escape codes")
 			} else {
 				assert.Check(t, !hasAnsi, "Output should not contain ANSI escape codes")
+				t.Logf("Output: %s", out)
```

Comment:
Could we make this part of the assert, instead of always logging?


```suggestion
assert.Check(t, !hasAnsi, "Output should not contain ANSI escape codes; out: %s", out)
```

Reply:
Done